doc/whats_new/v1.6.rst

```diff
@@ -71,12 +71,20 @@ more details.
   :class:`ensemble.StackingRegressor` now support metadata routing and pass
   ``**fit_params`` to the underlying estimators via their `fit` methods.
   :pr:`28701` by :user:`Stefanie Senger <StefanieSenger>`.
-
+  
 - |Feature| :class:`compose.TransformedTargetRegressor` now supports metadata
   routing in its `fit` and `predict` methods and routes the corresponding
   params to the underlying regressor.
   :pr:`29136` by :user:`Omar Salman <OmarManzoor>`.
 
+- |Feature| :class:`semi_supervised.SelfTrainingClassifier`
+  now supports metadata routing. The fit method now accepts ``**fit_params``
+  which are passed to the underlying estimators via their `fit` methods.
+  In addition, the `predict`, `predict_proba`, `predict_log_proba`, `score`
+  and `decision_function` methods also accept ``**params`` which are
+  passed to the underlying estimators via their respective methods.
+  :pr:`28494` by :user:`Adam Li <adam2392>`.
+
 Dropping official support for PyPy
 ----------------------------------
 
```

Comment:
I think we should mention the deprecated parameter in the changelog.

Reply:
Done.